components/compass-runtime-agent/cmd/init.go

```diff
@@ -12,15 +12,9 @@ import (
 	restclient "k8s.io/client-go/rest"
 	clientset "k8s.io/metrics/pkg/client/clientset/versioned"
 	"kyma-project.io/compass-runtime-agent/internal/apperrors"
-	"kyma-project.io/compass-runtime-agent/internal/k8sconsts"
 	"kyma-project.io/compass-runtime-agent/internal/kyma"
-	"kyma-project.io/compass-runtime-agent/internal/kyma/apiresources"
-	"kyma-project.io/compass-runtime-agent/internal/kyma/apiresources/accessservice"
```

Comment:
Please remove not used k8s resources clients (Istio, Secrets) stored in k8sResourceClientSets.

Reply:
Removed Istio resource client